docs/orchestrating-elastic-stack-applications/enterprise-search.asciidoc

```diff
@@ -15,6 +15,8 @@ This section describes how to deploy, configure and access Enterprise Search wit
 [id="{p}-enterprise-search-quickstart"]
 == Quickstart
 
+NOTE: Workplace Search requires an Elastic license. Give it a try by enabling an link:k8s-licensing.html[ECK trial license].
```

Comment:
```suggestion
NOTE: Workplace Search requires an Enterprise license. Give it a try with a link:k8s-licensing.html[trial license].
```

Just a suggestion, I'm definitely in favor of adding something here and don't feel too opinionated

I don't feel strongly either way

Reply:
> Workplace Search requires an Enterprise license

It is not _technically_ correct so I'm on the fence here.

Reply:
I would not introduce `Platinum` though in the context of ECK which would lead to more confusion.  Maybe we could have 
```suggestion
NOTE: Workplace Search requires an Enterprise license on ECK. Give it a try with a link:k8s-licensing.html[trial license].
```